spec/requests/admin/schools/cohorts/change_training_materials_spec.rb

```diff
@@ -43,22 +43,18 @@
 
   describe "POST /admin/schools/:school_slug/cohorts/:id/change-training-materials" do
     let(:core_induction_programme) { create(:core_induction_programme) }
-    let(:service_instance) { instance_double(Admin::ChangeInductionService) }
-
-    before do
-      allow(Admin::ChangeInductionService).to receive(:new).and_return service_instance
-      allow(service_instance).to receive(:change_cip_materials)
-    end
 
     it "calls change_cip_materials with the correct arguments" do
+      expect(Induction::ChangeCoreInductionProgramme).to receive(:call).with(
+        school_cohort: school_cohort,
+        core_induction_programme: core_induction_programme,
+      )
+
       put "/admin/schools/#{school.slug}/cohorts/#{cohort.start_year}/change-training-materials", params: {
         core_induction_programme_choice_form: {
           core_induction_programme_id: core_induction_programme.id,
         },
       }
-
-      expect(Admin::ChangeInductionService).to have_received(:new).with(school: school, cohort: cohort)
-      expect(service_instance).to have_received(:change_cip_materials).with(core_induction_programme)
```

Comment:
We may be loosing a bit of coverage here, are we no longer calling the `change_cip_materials` with `core_induction_programme`?

Reply:
These tests are only testing to see if the service classes are being called correctly.
In this case, we no longer receive that call inside the controller

Reply:
also you want to change the spec title

Reply:
Do we really need to do that? change_cip_materials is something specific to Admin::ChangeInductionService and not Induction::ChangeCoreInductionProgramme

Reply:
@StupidCodeFactory 
that part of the code has been removed:
https://github.com/DFE-Digital/early-careers-framework/pull/2171/files#diff-8f61d04c0c5f1bb6bbec1a583c63c7aceabc77c7fd708ca45b8d27b8c6e09390R28